app/utils/fileUpload/interfaces.ts

```diff
@@ -0,0 +1,7 @@
+export interface IFile {
```

Comment:
What the difference between `IFile` and `IAttachment` ?

Reply:
IFile is just for files that we will upload! IAttachment is for any file that we are receiving!
Then, the params aren't the same.

Reply:
I think we can change this `IFile` to `IFileUpload` and trasform to a definitions. Because we have the same interface at: `app/views/ShareListView`